gm_dotnet_managed/GmodNET/Startup.cs

```diff
@@ -156,7 +156,10 @@ internal static IntPtr Main(IntPtr lua_base, IntPtr native_version_string, int v
                     LuaInterop.check_number = CreateNativeCaller<Func<IntPtr, int, double>>(params_from_native_code[53]);
 
                     LuaInterop.push_c_function_safe = CreateNativeCaller<Action<IntPtr, IntPtr, IntPtr>>(params_from_native_code[54]);
+
+                    GmodInterop.lua_extructor = &LuaInterop.ExtructLua;
```

Comment:
Wait, do you mean Extract rather than Extruct?

Reply:
Oops, will fix it now 😂